src/pages/thanks.tsx

```diff
@@ -0,0 +1,14 @@
+import { Box } from "@mui/system";
+import React from "react";
+import Thanks from "../components/Thanks";
+
+const thanks = () => {
+  return (
+    <div className="main">
+      <Thanks></Thanks>
+      <Box sx={{ height: "2px" }}>test</Box>
```

Comment:
and maybe delete this! I added it before for just testing footer so you can delete it!

Reply:
yeah, I fix it :)